viadot/config.py

```diff
@@ -36,15 +36,15 @@ def from_yaml(cls, path: str, key: Optional[str] = None) -> Config:
         CONFIG = Config()
 
 
-def get_source_config(key):
-    source_configs = CONFIG.get("sources")
+def get_source_config(key, config):
```

Comment:
`config` should be optional as otherwise you would break all existing sources that use this function

Reply:
you can set `CONFIG` as default

Reply:
Done